src/OLEDDisplay.cpp

```diff
@@ -801,15 +810,20 @@ int OLEDDisplay::_putc(int c) {
 // Private functions
 void OLEDDisplay::setGeometry(OLEDDISPLAY_GEOMETRY g, uint16_t width, uint16_t height) {
   this->geometry = g;
+
   switch (g) {
   	case GEOMETRY_128_64:
     	this->displayWidth = 128;
     	this->displayHeight = 64;
 		break;
-	case GEOMETRY_128_32:
+	  case GEOMETRY_128_32:
     	this->displayWidth = 128;
     	this->displayHeight = 32;
 		break;
+    case GEOMETRY_64_48:
```

Comment:
Can you please fix the formatting in this `switch` statement (indentation), thanks.

Reply:
could you please approve this change request? it has been opened since 2018 and it basically works...

Reply:
I approved it a year ago and we would have merged it if all feedback had been addressed or if we had been given permission to edit the file in question ourselves.

Reply:
Hey Marcel, 
thanks for replying. 
Isn't the only todo open: to fix the indenting of one code block?

Cheers
l.